### PR TITLE
Some QOL changes to output files.  Blocks outputs in same select call…

### PR DESCRIPTION
**Pull request by fdaniels110**
…. Also stops timestamp duplication that was happening.

### CODE REVIEW

**Comment by flip40** [comment]
Time stamp duplication was intentional, it is part of the homework. While it won't show up due to the average CPU being far to fast, it is a time for when the parent printed the message, as opposed to the time when the child wrote the message to the pipe. Regardless, the homework description mentions having these two time stamps.

**Comment by fdaniels110** [comment]
Ah my bad, I thought that only one was necessary.  I'll delete those changes. Thought the double type change and the return for that function is something I would want changed as well as moving the newline char from the printf statement.

**Comment by flip40** [comment]
Have you been able to compile and test this code?

**Comment by fdaniels110** [comment]
Now it compiles. I just saw that the print statements will need to be changed to file writers though. I'll try to finish it today.

File: main.c [hunk 1]
```diff
@@ -191,9 +192,10 @@ int main(int argc, char const *argv[]) {
 						char readbuffer[BUFFER_SIZE];
 						memset(readbuffer, '\0', sizeof(readbuffer));
 						read(pipes[i], readbuffer, sizeof(readbuffer)-1);
-						if (readbuffer[0] != '\0') printf("%5.3lf %s\n", get_current_time() - start_time, readbuffer);
+						if (readbuffer[0] != '\0') printf("%5.3lf %s",get_current_time() - start_time, readbuffer);
 					}
 				}
+				printf("\n");
```
Comment (flip40) [inline main.c]: This is in the wrong place, causing all children to append onto the same line. It should be with the other print statement:
```
if (readbuffer[0] != '\0') {
	printf("%5.3lf %s",get_current_time() - start_time, readbuffer);
	printf("\n");
}
```

**Comment by fdaniels110** [comment]
lets just pretend this branch never happened as I already finished the stdin portion and everything works.

**Pull request closed without merging**